spec/views/users/users_show_spec.rb

```diff
@@ -19,8 +19,31 @@
       users.each { |user| expect(page).to have_css("img[src='#{user.photo}']") }
     end
 
+    it 'should display user name' do
+      users.each { |user| expect(page).to have_content(user.name) }
+    end
+
     it 'should display the number of posts' do
       users.each { |user| expect(page).to have_content("Number of posts: #{user.posts.count}") }
     end
+
+    it 'should display user bio' do
+      users.each { |user| expect(page).to have_content(user.bio) }
+    end
+
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to implement tests that assert you can see a button that lets me view all of a user's posts.